### PR TITLE
Zabbix returner

### PR DESCRIPTION
**Pull request by FerrySchuller**
### What does this PR do?

Created a returner for Zabbix
### What issues does this PR fix or reference?

New feature
### Tests written?

No

### CODE REVIEW

File: salt/returners/zabbix_return.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+def __virtual__():
+    return __virtualname__
+
+
+def zabbix_sender(key, host, output):
+    zabbix_sender = "/usr/local/zabbix/bin/zabbix_sender"
+    zabbix_config = "/usr/local/zabbix/etc/zabbix_agentd.conf"
+    cmd = zabbix_sender + " -c " + zabbix_config + " -s " + host + " -k " + key + " -o \"" + output +"\""
+    os.system(cmd)
```
Comment (cachedout) [inline salt/returners/zabbix_return.py]: Using `__salt__['cmd.run']` here might be a better option.

**Comment by cachedout** [comment]
Hi @FerrySchuller 

Just one suggestion in-line and then some lint errors to fix up. Thanks! 

https://jenkins.saltstack.com/job/PR/job/salt-pr-lint-n/2145/violations/file/salt/returners/zabbix_return.py/

File: salt/returners/zabbix_return.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+# Define the module's virtual name
+__virtualname__ = 'zabbix'
+
+
+def __virtual__():
+    return __virtualname__
+
+
+def zabbix_sender(key, host, output):
+    zabbix_sender = "/usr/local/zabbix/bin/zabbix_sender"
```
Comment (kev009) [inline salt/returners/zabbix_return.py]: You should probably let these both be overridden from __opts__

File: salt/returners/zabbix_return.py (hunk 1 above)
Comment (FerrySchuller) [inline salt/returners/zabbix_return.py]: Fixed.

File: salt/returners/zabbix_return.py (hunk 2 above)
Comment (cachedout) [inline salt/returners/zabbix_return.py]: I agree here though it's fine to submit a follow-up PR to do this.

**Comment by cachedout** [comment]
@FerrySchuller Just a minor lint fix-up here: https://jenkins.saltstack.com/job/PR/job/salt-pr-lint-n/2252/violations/file/salt/returners/zabbix_return.py/

**Pull request merged**